src/rendering/renderers/shared/system/SharedSystems.ts

```diff
@@ -25,14 +28,17 @@ export const SharedSystems = [
     TextureGCSystem,
     GenerateTextureSystem,
     ExtractSystem,
+    HTMLTextSystem
 ];
 
 export const SharedRenderPipes = [
     BlendModePipe,
     BatcherPipe,
     SpritePipe,
+    NineSliceSpritePipe,
     RenderGroupPipe,
     AlphaMaskPipe,
     StencilMaskPipe,
     ColorMaskPipe,
+    TilingSpritePipe,
```

Comment:
This gets added in the init.ts for tiling sprites, so shouldn't be added here

Reply:
ta, removed!